src/vif.jl

```diff
@@ -26,6 +26,7 @@ function get_matrix_term(x)
     x = collect_matrix_terms(x)
     x = x isa MatrixTerm ? x : first(x)
     x isa MatrixTerm || throw(ArgumentError("couldn't extract matrix term from $x"))
+    # this is a work around for some weirdness that happens in MixedModels.jl
```

Comment:
Is there a way to test this here? It doesn't seem covered.

Reply:
tested by manually constructing such a MatrixTerm.

I don't want to take MixedModels (or GLM for that matter) as a test dependency because then we'll have trouble testing breaking releases because we won't be able to install the reverse dependencies.